news/history-all.rst

```diff
@@ -0,0 +1,50 @@
+**Added:** None
```

Comment:
Ahh, also you should remove the `None` here or none of the contents below will get rolled in to the changelog